index.html

```diff
@@ -9,11 +9,37 @@
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
     <link rel="stylesheet" href="./style.css">
 </head>
-
     <main>
         <div class="container-lg ">
+
+          <nav class="navbar navbar-light bg-light  border border-dark">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="#" id="home">
+      
+      Awesome Books
+    </a>
+
+    <div class ="">
+     <ul class="nav">
+  <li class="nav-item">
+    <a class="nav-link active" aria-current="page" href="#" id="list">List</a>
+  </li>
+  <li class="nav-item">
```

Comment:
- Good job so far trying to implement the localstorage, but its implementation does not save more than one book, it keeps changing what was previously in the storage and on refresh all list on the webpage disappears, Please try to fix it.

You can reach out to me on slack via `@kanu mike chibundu`